src/test/java/com/synacy/lesson01/exercises/ProductTest.java

```diff
@@ -0,0 +1,18 @@
+package com.synacy.lesson01.exercises;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ProductTest {
+    @Test(expected = IllegalArgumentException.class)
+    public void product_shouldThrowIllegalArgumentException() throws Exception {
```

Comment:
I was thinking that maybe a negative value would have been an illegal argument for the function